os/arch/Kconfig

```diff
@@ -735,34 +736,60 @@ config RAM_VSTART
 config RAM_SIZE
 	int "Primary RAM size"
 	default 0
+	depends on MM_NHEAPS = 1
 	---help---
 		The size in bytes of the installed primary RAM. "Primary" RAM
 		refers to the RAM that you link program code into.  If program code
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
+		This configuration can be used only MM_NHEAPS sets to 1.
 
 config RAMx_START
+	string "List of address for multiple heap memory"
+	default "0x00000000,"
+	depends on MM_NHEAPS != 1
+	---help---
+		The address list of the multiple heap memory.
+		When you want to use multi-heap in a RAM, set MM_NHEAPS first at memory
+		management, and then set hexa values at RAMx_START and RAMx_SIZE
+		to support multi-heap memory.
+		Each address is separated by ',' as shown in example.
+		i.e. "0x02000000,0x4000000"
+
+config RAMx_SIZE
+	string "List of size for multiple heap memory"
+	default "0,"
+	depends on MM_NHEAPS != 1
+	---help---
+		The size list of multi-heap memory.
+		Refer RAMx_START content. One different thing is this has
+		decimal values (START has hexa values).
+		This should have same number of list element with RAMx_START's.
+		Each size is separated by ',' as shown in example.
+		i.e. "100,200"
+
```

Comment:
Is there any condition check while initializing the heaps? i.e. whether the heap size is within the size range of region or not .